plasma-contract.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:8.16-alpine
+
+MAINTAINER OmiseGO Engineering <eng@omise.co>
+
+WORKDIR /home/node
+
+RUN apk add --update \
+    python \
+    python-dev \
+    py-pip \
+    build-base \
+		git
+
+RUN git clone https://github.com/omisego/plasma-contracts.git
```

Comment:
I think we'll need to focus on SHA from the contracts as @pdobacz said.

Reply:
Locked this to dc3d60a72b814916902c343cea30e65f1e4144c3 from `plasma-contracts`. See f9435faf87a881c8a3a9f83de76b411ce8c569a2